misc/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM ubuntu:latest
+COPY ./model-csi-driver /usr/bin/model-csi-driver
```

Comment:
Please move the Dockerfile to the `build` directory, refer to https://github.com/golang-standards/project-layout/tree/master/build.